cmd/list.go

```diff
@@ -48,16 +49,28 @@ const (
 	rowMaxHeight = 1
 )
 
+type (
+	ListOptions struct {
+		*options.Options
+		Tags []string
+	}
+)
+
 func listCmd() *cobra.Command {
 	documentIDs := []string{}
 
+	opts := &ListOptions{
+		Options: options.New(options.WithLogger(utils.NewLogger("list"))),
+	}
+
 	listCmd := &cobra.Command{
 		Use:     "list [flags] SBOM_ID...",
 		Aliases: []string{"ls"},
 		Short:   "List SBOM documents in local cache",
 		Long:    "List SBOM documents in local cache",
 		PreRun: func(_ *cobra.Command, args []string) {
 			documentIDs = append(documentIDs, args...)
+			preRun(opts.Options)
```

Comment:
Use `args` directly instead of the separate `documentIDs` variable, so `listCmd.PreRun` just becomes

```suggestion
		PreRun: preRun(opts.Options),
```